File: Examples/C/Hello.c
// Copyright (c) TribuFu. All Rights Reserved

#include "stdio.h"

#include "Library.h"

extern float Add(float Num1, float Num2);
extern float Divide(float Num1, float Num2);
extern float Multiply(float Num1, float Num2);
extern float Subtract(float Num1, float Num2);

int main()
{
    printf("Rust Calculator\n");

    int num1 = 1;
    int num2 = 2;

    float added = Add(num1, num2);
    float subtracted = Subtract(num1, num2);
    float multiplied = Multiply(num1, num2);
    float divided = Divide(num1, num2);

    printf("\nAdded: %f", added);
    printf("\nSubtracted: %f", subtracted);
    printf("\nMultiplied: %f", multiplied);
    printf("\nDivided: %f", divided);

    while (1)
    {
    }

    system("PAUSE");
}

File: Examples/Rust/Hello.rs
#![allow(non_snake_case)]

fn main() {
    let num1 = 1.0;
    let num2 = 2.0;

    println!("Added: {}", Library::Add(num1, num2));
    println!("Subtracted: {}", Library::Subtract(num1, num2));
    println!("Multiplied: {}", Library::Multiply(num1, num2));
    println!("Divided: {}", Library::Divide(num1, num2));
}
